taxcalc/dropq/dropq_utils.py

```diff
@@ -116,9 +115,9 @@ def dropq_calculate(year_n, start_year,
             calc1p.increment_year()
         calc1p.calc_all()
         assert calc1p.current_year == start_year
-        # compute mask that shows which of the calc1 and calc1p results differ
-        # mask is true if a filing unit's tax liability changed after a dollar
-        # was added to the filing unit's income
+        # compute mask showing which of the calc1 and calc1p results differ;
+        # mask is true if a filing unit's incom tax liability changed after
```

Comment:
Here's a typo. You left the "e" off of income.

Reply:
@hdoupe,  Thanks for catching this.  I appreciate the close review.